### ISSUE TITLE
Support running behat tests against an instance hosted on prem

### ISSUE THREAD

**Issue by jwfuller** [CuBoulder/express#1982]
# Goals

- Basic validation of infrastructure (verify that Express features work)
- Poor man's load testing (generate load from the front end)
  - Might be able to use https://robo.li/tasks/Base/#parallelexec and https://robo.li/tasks/Testing/#behat to run several test suites in parallel

# Proposed direction

- Move creation of users and testing setup out of the hosting modules

# Notes

- Should be able to us the following modification to the Mink Extension config
```
Behat\MinkExtension:
        base_url: "https://www-dev-https.colorado.edu"
```

**Comment by alexfinnarn** [comment]
The easiest way to maintain changes in the config, like base_url, is probably through another config file. If several options are different between test runs, then you end up with long variable exports. It is far easier to read the YAML files that modify a BEHAT_PARAMS variable.

https://github.com/Behat/Behat/pull/1048/files

A PR was merged into the Behat project but then removed that appears to allow environmental variables in YAML. Since it is only a one-line change, trying out that patch might be worthwhile. 

Otherwise, https://github.com/CuBoulder/express/issues/1969 should allow for user prompts instead of manually changing a YAML file or exporting variables before running the test command.
